TestCases/cont_adj_euler/wedge/of_grad_combo.dat.refdiscrete

```diff
@@ -1,5 +1,5 @@
 VARIABLES="VARIABLE"      , "GRADIENT"      , "FINDIFF_STEP"  
-          0               , 0.00146785      , 0.001           
-          1               , 0.0015213       , 0.001           
-          2               , 0.00177652      , 0.001           
-          3               , 0.00222003      , 0.001           
+          0               , 0.00651231      , 0.001           
```

Comment:
Do you know why these have changed?

Reply:
Yes, in fact I had to chase this one down for a bit. In principle this is a bug in the current code. It comes because the adaptive CFL is enabled and any time we do a recording after the first one (so, after there has been any geometry recording, for example) it uses the adapted CFL, so the path is changed a bit. With this version, that's no longer the case as the CFL adaptation is not called for the adjoint driver.